modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/AdminTooltip.js

```diff
@@ -92,36 +123,49 @@ const AdminTooltip = ({
 					<label htmlFor={`${namespace}pin-sku`}>
 						{Liferay.Language.get('select-sku')}
 					</label>
-					<ClayAutocomplete>
+
+					<ClayAutocomplete ref={node}>
 						<ClayAutocomplete.Input
-							onChange={(event) => setSku(event.target.value)}
-							value={sku}
-						/>
-						<ClayAutocomplete.DropDown
-							active={skus}
-							onSetActive={(event) => {
-								setSku(
-									event.target.innerText.match(/^[\S]*/g)[0]
-								);
+							onChange={(event) => {
+								setSelectedProduct(null);
+
+								setQuery(event.target.value);
+							}}
+							onFocus={(_e) => {
```

Comment:
Since `_e` is unused, you can just remove it. We use underscores to omit unused variables only when we need access to subsequent arguments.

For example `(_a, b) => b.toString()`.